tests/integration/bootstrap/RestContext.php

```diff
@@ -279,9 +279,9 @@ public function iRequest($pageUrl)
 
 		// Get response object
 		$this->response = $response;
-		// $data = $this->response->getBody(true);
-		// $data = explode("\n", $data);
-		// \Log::info(print_r($data, true));
+		$data = $this->response->getBody(true);
+		$data = explode("\n", $data);
```

Comment:
I'm not sure I understand why we do this here for the print r?

Reply:
oh sorry that should be commented out, I forgot to revert it